### ISSUE TITLE
Body clips with default page width before 875px breakpoint

### ISSUE THREAD

**Issue by numberoverzero** [sphinx-doc/alabaster#54]
While working on #44 I discovered that document text gets clipped under ~911 px but the nav doesn't move until 875px.

I uploaded [three images](http://imgur.com/a/A7KRN) to show off the problem with the current breakpoints and the default theme width.

Where are the 875/870px numbers from for the media queries?

**Comment by bitprophet** [comment]
Thanks for the report! Confirming that I can recreate this on my end, huh. It goes by real fast if you resize as fast as I normally do when spot checking the smaller width layouts, which is one reason why it's gone uncaught so far.

As with everything else in here, those specific numbers come from the precursor themes we're based on, so I dunno exactly how they were calculated. I just checked and e.g. http://python-requests.org (which as far as I can tell is still using Kenneth's original theme) has the same bug. So it's probably been around for some time & may not be due to any changes we've made on our end.

Open to suggestions for a fix, don't have time to dig right now for something this low priority :)
